### PR TITLE
utils_misc: Add support for case rh_qemu_iotests

### PR DESCRIPTION
**Pull request by pingl2016**
1. Provide API to get the path of qemu-nbd binary for use in tests
2. No need to check the image for case rhe_qemu_iotests

Signed-off-by: Ping Li <pingl@redhat.com>

ID: 1582035

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2296,38 +2296,35 @@ def get_qemu_dst_binary(params):
...
-    if not os.path.isfile(qemu_img_binary_path):
-        logging.debug('Could not find params qemu-img in %s, searching the '
-                      'host PATH for one to use', qemu_img_binary_path)
-        qemu_img_binary = utils_path.find_command('qemu-img')
-        logging.debug('Found %s', qemu_img_binary)
-    else:
-        qemu_img_binary = qemu_img_binary_path
-
-    return qemu_img_binary
+    return get_binary('qemu-img', params)
```
Comment (hereischen) [inline virttest/utils_misc.py]: Gievn `qemu-img`, `params.get("%s_binary" % binary_name, binary_name)` would be `qemu-img_binary`.

Comment (pingl2016) [reply virttest/utils_misc.py]: Good Catch. Updated, thanks

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by xutian** [comment]
looks OK to me.

**Pull request merged**